src/components/TagsPoplist/index.tsx

```diff
@@ -60,6 +61,11 @@ type TagsPoplistProps = {
   multiline?: boolean
 }
 
+const TagContainer = styled.div<{ index: number; visibleTagsCount: number }>`
+  margin-right: ${({ index, visibleTagsCount }) =>
+    index + 1 !== visibleTagsCount ? 1 : 0};
+`
```

Comment:
nope, because we don't have the parameters `index`, `visibleTagsCount` in the parent what we could do is add div and the style on it which i think its  so over, what do you think?

Reply:
Well if you put a gap on the parent you won't need to have index and visibleTagsCount no ? 

You already have StyledTagContainer as parent